libs/designsystem/src/lib/components/list/list.component.scss

```
@@ -241,3 +242,10 @@ ion-item-options[side='end'] {
     }
   }
 }
+
+@each $sizeName, $size in utils.$sizes {
+  .stand-alone-item-group.#{$sizeName},
+  .stand-alone-list.#{$sizeName} {
+    margin-bottom: $size;
+  }
+}
```

This looks like a nicer way to solve it. Be aware that the increased specificity (2 x class selector) makes it a bit harder to override. May or may not be a problem.

On the other hand, `.stand-alone-item-group` and `.stand-alone-list` appear to be unnecessary. They seem to only be used here in combination with `sizeName`. You could consider to not use them. This will simplify things a bit _and_ decrease specificity.

You'll have to update the template file accordingly.

```suggestion
@each $sizeName, $size in utils.$sizes {
  .#{$sizeName} {
    margin-bottom: $size;
  }
}
```

---

That makes a lot of sense :) I'm on it!